### ISSUE TITLE
Snow on contraption

### ISSUE THREAD

**Issue by jolinar927** [ValkyrienSkies/Valkyrien-Skies-2#1018]
### Mod Name

Primal Winter

### This issue occurs when only Valkyrien Skies, addons, and the mod I have specified are installed and no other mods

- [X] I have tested this issue and it occurs with only Valkyrien Skies, addons, and the mod I have specified

### Minecraft Version

1.20.x

### Mod Loader

Forge

### Issue description

I use Primal Winter, which allows winter and snowstorms in every biome, except for those that are blacklisted, the default setting of the mod makes the End and the Nether completely snowy everywhere. Whenever I build a VS2 contraption in the Nether (where it doesn’t snow), the vehicle starts becoming snowy.
I also have VS Tournament, Interact me and VS Addition.

![Minecraft_ Forge 1 20 1 - Singleplayer 2024  12  06  19_03_02](https://github.com/user-attachments/assets/f6a00fee-7ce4-4bad-bac4-f6f7eb401828)

### Issue reproduction

Install Primal Winter (https://www.curseforge.com/minecraft/mc-mods/primal-winter) and VS2, build a contraption in the Nether, and over time, it will become snowy.

**Comment by thetheaplant** [comment]
yes, the mod which allows snow in any biome is allowing snow in any biome

**Comment by jolinar927** [comment]
Snow is not allowed in the nether, and the contraption was assembled there.

**Comment by thetheaplant** [comment]
The ships are stored in the overworld either way, consider setting the max snow accumulation height to 0. This likely won't be fixed any time soon unless a third party fixes it.

**Comment by jolinar927** [comment]
Sure, thank you for your answer and your time!

**Comment by Rubydesic** [comment]
The ships aren't stored in the Overworld, they're always stored in the same dimension. If the blacklist is for the entire Nether and all nether biomes it should work.
